go-controller/pkg/node/secondary_node_network_controller_test.go

```diff
@@ -0,0 +1,71 @@
+package node
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/factory"
+	ovntest "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/testing"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+)
+
+var _ = Describe("BaseSecondaryNetworkController", func() {
```

Comment:
```suggestion
var _ = Describe("SecondaryNodeNetworkController", func() {
```
? seems like you are using SecondaryNodeNetworkController not base for this test based on file naming and controller used